README.md

```diff
@@ -51,15 +51,35 @@ All documentation and the issue tracking can be found in our main repository her
 | Smart Charging             | :heavy_check_mark: yes    |
 | Remote Trigger             | :heavy_check_mark: yes    |
 
-| Whitepapers                                                                                                                               | Supported              |
+| Whitepapers & Application Notes                                                                                                                              | Supported              |
 | ----------------------------------------------------------------------------------------------------------------------------------------- | ---------------------- |
 | [OCPP 1.6 Security Whitepaper (3rd edition)](https://www.openchargealliance.org/uploads/files/OCPP-1.6-security-whitepaper-edition-3.zip) | :heavy_check_mark: yes |
 | [Using ISO 15118 Plug & Charge with OCPP 1.6](https://www.openchargealliance.org/uploads/files/ocpp_1_6_ISO_15118_v10.pdf)                | :heavy_check_mark: yes                    |
+| [OCPP & California Pricing Requirements](https://www.openchargealliance.org/uploads/files/ocpp_and_dms_evse_regulation-v2.0.pdf)                |                               |
 
 ### Support for OCPP 2.0.1
 
-The development of OCPP2.0.1 has already started and is in progress. See also the general roadmap of EVerest:
-https://github.com/EVerest/EVerest/blob/main/tsc/ROADMAP.md
+The development of OCPP2.0.1 is in progress. 
+[For details, see the sheet with the current implementation status.](https://docs.google.com/spreadsheets/d/1NXEa7d0pkT4_1L9lZ9RpiLHGPtTKoqwPsKLJPjTZ_vM/edit?usp=sharing)
+
+### Feature Profile Support OCPP 2.0.1
+
+| Feature Profile            | Supported                 |
+| -------------------------- | ------------------------- |
+| Core                       | WIP                       |
```

Comment:
isnt WIP to weak? since its already in certification? in a way SW is ALWAYS WIP, so I think we reached that point here as well? so maybe say yes? at least thats what we communicating on slides.
smart charging would be WIP  if I get JOET correctly..

Reply:
Have just changed both.

I suggest to publish that already as this documentation will all in all be a dynamically changing one in the next weeks / months.

Reply:
@caller Seems like you will have to approve this change (or the whole PR) before we can merge it.